lib/rogue/models/player_phase.rb

```diff
@@ -10,31 +10,21 @@ def initialize(game: Rogue.game, last_action: nil)
 
   attr_accessor :last_action
 
+  DIRECTION_KEYS = [:h, :u, :k, :i, :l, :m, :j, :n]
```

Comment:
I don't think that this DRYs up the code at all since we have to still specify which keys map to which direction and the code depends on the order of the keys. I'd prefer being able to see which keys map to which action without having to look at the coordinate class, I think it's less error prone that way.